V2/Makefile

```diff
@@ -3,6 +3,7 @@ help:
 		@echo "---------------HELP-----------------"
 		@echo "To install the dependencies type make install"
 		@echo "To test the project type make test"
+		@echo "To test a specific test of the project make test specific_test=name of test"
```

Comment:
Maybe we can improve this help:
"- make install --> Install the dependencies"
"- make test --> Run all tests"
"- make test specific_test=<name_of_the_test> --> Run specific test"
or something like that